src/databricks/labs/ucx/queries/assessment/main/06_4_submit_runs.sql

```diff
@@ -0,0 +1,8 @@
+-- viz type=table, name=Submit Runs, columns=hashed_id,failure,run_ids
+-- widget title=Incompatible Submit Runs, row=6, col=4, size_x=3, size_y=8
+SELECT
+    hashed_id,
+    EXPLODE(FROM_JSON(failures, 'array<string>')) AS failure,
+    FROM_JSON(run_ids, 'array<string>') AS run_ids
+FROM $inventory.submit_runs
+ORDER BY hashed_id DESC
```

Comment:
add another viz, where you group by failure and count hashed_ids

Reply:
done